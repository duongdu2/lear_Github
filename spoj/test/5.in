3
9
98 16 90 72 87 4 99 75 11 
6
68 64 71 99 29 22 
8
20 5 4 15 27 47 86 76 
