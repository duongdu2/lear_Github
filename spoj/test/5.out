287
238
209
